tools/python/liveanalytics_migration_scripts/targets/rds_for_postgresql/README.md

```diff
@@ -53,19 +49,19 @@ python copy_postgres.py \
 
 - `--table`: Target PostgreSQL table name
 - `--database`: PostgreSQL database name
-- `--csv_files_dir`: Directory or glob pattern for CSV files
+- `--csv-files-dir`: Directory for CSV files
 - `--host`: Aurora/RDS Postgres cluster writer endpoint
-- `--secret_name`: AWS Secrets Manager secret containing database password
+- `--secret-arn`: ARN of AWS Secrets Manager secret containing database password
```

Comment:
I updated this to be optional and if the arn is omitted, users can enter the password through standard input. This way we can support self managed credentials.